docker/compose.yml

```diff
@@ -0,0 +1,143 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+services:
+  minio:
+    image: minio/minio:latest
+    environment:
+      MINIO_ROOT_USER: minio
+      MINIO_ROOT_PASSWORD: minio_minio
+      MINIO_DEFAULT_BUCKETS: spark-bucket
+    hostname: minio
+    ports:
+      - 9000
+      - 9001:9001
+
+  mysql:
+    image: mysql:${MYSQL_VERSION}-oracle
+    environment:
+      MYSQL_USER: mysql
+      MYSQL_PASSWORD: mysql
+      MYSQL_DATABASE: metastore
+      MYSQL_ALLOW_EMPTY_PASSWORD: TRUE
+    hostname: mysql
+    ports:
+      - 3306
+    volumes:
+      - ./image/hive-schema-2.3.0.mysql.sql:/docker-entrypoint-initdb.d/hive-schema-2.3.0.mysql.sql
+      - ./image/hive-txn-schema-2.3.0.mysql.sql:/docker-entrypoint-initdb.d/hive-txn-schema-2.3.0.mysql.sql
+
+  zookeeper:
+    image: zookeeper:${ZOOKEEPER_VERSION}
+    ports:
+      - 2181
+    hostname: zookeeper
+
+  clickhouse-s1r1:
+    image: clickhouse/clickhouse-server:${CLICKHOUSE_VERSION}
```

Comment:
Can you update the clickhouse version to a newer one ?
It seems they have added arm64 Docker images just recently - https://hub.docker.com/r/clickhouse/clickhouse-server/tags

Reply:
Sounds good, let me switch to 22.3.3.44

Reply:
I just tried with [22.3.3.44](https://hub.docker.com/layers/clickhouse-server/clickhouse/clickhouse-server/22.3.3.44/images/sha256-171b386a1ae34ae0192faa751579a31334c15bea0ed57b5ca907937696db25d4?context=explore) but it fails with:

```
docker-clickhouse-s1r1-1  | /entrypoint.sh: line 39:    13 Segmentation fault      $gosu test -f "$CLICKHOUSE_CONFIG" -a -r "$CLICKHOUSE_CONFIG"
docker-clickhouse-s1r1-1  | Configuration file '/etc/clickhouse-server/config.xml' isn't readable by user with id '101'
```

Reply:
Look unreasonable, `config.xml` is granted to read by anyone.
```
(scc) ➜  spark-clickhouse-connector git:(docker) ll clickhouse-core/src/testFixtures/conf/clickhouse-cluster/config.xml
-rw-r--r--  1 chengpan  staff   4.6K Mar 15 18:21 clickhouse-core/src/testFixtures/conf/clickhouse-cluster/config.xml
```

Reply:
I confirm 22.3.3.44 works well on my x86 macOS

Reply:
I will investigate it now!